### PR TITLE
Valeriyapetrova/hometask 1

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: valeriyapetrova/hometask-1/hello_world.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env ruby
```
Comment (TanyaAkulich) [inline valeriyapetrova/hometask-1/hello_world.rb:1]: pulled to test locally -- file is executable
maybe CI issue remotes/origin/valeriyapetrova/hometask-1

**Review by TanyaAkulich — DISMISSED** [review]

**Comment by TanyaAkulich** [comment]
@ramantechart can we merge it?

**Review by ramantechart — CHANGES_REQUESTED** [review]
Please make sure to sync the fork with original repo

**Review by TanyaAkulich — APPROVED** [review]
synced 👍

**Review by ramantechart — APPROVED** [review]

**Pull request merged**